Documentation/Manual/Texinfo/chap-inspector.texi

```diff
@@ -169,23 +428,22 @@ sample"))
 
 Next, we want to be able to compute the mean. This is just the standard
 average that everyone learns: add up all the numbers and divide by how
-many of them there are. It's written @math{\overline {x}}
+many of them there are. It's written @math{\overline {x}}.
 
 @lisp
 (defgeneric mean (sample)
   (:documentation "The mean of the numbers in a statistical
 sample"))
 
 (defmethod mean ((sample sample))
-  (/ (sum sample)
-     (sample-size sample)))
+  (/ (sum sample) (sample-size sample)))
 @end lisp
 
 Finally, to be really fancy, we'll throw in a function to compute the
-standard deviation. You don't need to understand this, but the standard
-deviation is a measurement of how spread out or bunched together the
-numbers in the sample are. It's called @math{s}, and it's computed like
-this: @math{s = \sqrt{{1 \over N-1} \sum_{i=1}^N (x_i - \overline {x})^2}}
+standard deviation which is a measurement of how spread out or bunched
+together the numbers in the sample are. It's called @math{s}, and it's
+computed like this: @math{s = \sqrt{{1 \over N-1} \sum_{i=1}^N (x_i -
+\overline {x})^2}}.
```

Comment:
could use a rendered png for formula here. currently math doesn't render in HTML version.

Reply:
I was thinking about trying [MathJax](https://www.mathjax.org/) to solve this properly, but I think that is beyond the scope of this pull request.

Reply:
No hurry with this. also look into https://github.com/stylewarning/formulador for .info version.